### PR TITLE
Split "Context" textarea field inside Issue template into several required input fields

### PR DESCRIPTION
**Pull request by evgfilim1**
Example: https://github.com/evgfilim1/actions-test/issues/7

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 1]
```diff
@@ -12,16 +12,31 @@ body:
...
+    attributes:
+      label: Python version
+      placeholder: e.g. 3.10.1
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: aiogram version
+      placeholder: e.g. 2.21 or 3.0b3
```
Comment (uwinx) [inline .github/ISSUE_TEMPLATE/bug.yaml:39]: Maybe add `options` enumeration for major revision? 2.x / 3.x and allow specifying x (the rest of the revision: minor, patch, etc.)

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 2]
```diff
@@ -12,16 +12,31 @@ body:
...
+    attributes:
+      label: Operating system
+      placeholder: e.g. Ubuntu 20.04.2 LTS
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Python version
+      placeholder: e.g. 3.10.1
```
Comment (uwinx) [inline .github/ISSUE_TEMPLATE/bug.yaml:32]: Maybe add `options` enumeration for supported versions?

File: .github/ISSUE_TEMPLATE/bug.yaml (hunk 1 above)
Comment (evgfilim1) [reply .github/ISSUE_TEMPLATE/bug.yaml:39]: I don't understand the rationale behind it. A user will anyway write a version inside a text field.

File: .github/ISSUE_TEMPLATE/bug.yaml (hunk 2 above)
Comment (evgfilim1) [reply .github/ISSUE_TEMPLATE/bug.yaml:32]: This may be useful, but again, maintaining a list of all possible `major.minor` versions will be a bit hard.

**Pull request merged**